src/js/tech/html5.js

```diff
@@ -728,32 +728,35 @@ class Html5 extends Tech {
       return super.addRemoteTextTrack(options);
     }
 
-    var track = document.createElement('track');
+    var htmlTrackElement = document.createElement('track');
```

Comment:
There is a mixed bag between track, trackElement, htmlTrackElement, textTrackEls, etc (Tried my best to follow existing convention with adding a bit more specificity).